tests/upgrades/test_errata.py

```diff
@@ -0,0 +1,260 @@
+        rhel_repo = entities.Repository(product=product,
+                                        content_type='yum',
+                                        url=rhel_repo_url,
+                                        ).create()
+        call_entity_method_with_timeout(rhel_repo.sync, timeout=1400)
+        return tools_repo, rhel_repo
+
+    @pre_upgrade
+    def test_pre_scenario_generate_errata_for_client(self):
+        """Create product and repo from which the errata will be generated for the
+        Satellite client or content host
+
+        :id: 88fd28e6-b4df-46c0-91d6-784859fd1c21
+
+        :steps:
+
+            1. Create  Life Cycle Environment, Product and Custom Yum Repo
+            2. Create custom tools, rhel repos and sync them
+            3. Create content view and publish it
+            4. Create activation key and add subscription.
+            5. Registering Docker Content Host RHEL7
+            6. Check katello agent and goferd service running on host
+            7. Generate Errata by Installing Outdated/Older Packages
+            8. Collect the Erratum list
+
+
+        :expectedresults:
+
+            1. The content host is created
+            2. errata count, erratum list will be generated to satellite client/content host
+
+        """
+        client_os = DISTRO_DEFAULT
+        # Step 1: Create  Life Cycle Environment, Product and Custom Yum Repo
+        environment = entities.LifecycleEnvironment(
+            organization=self.org
+        ).search(query={'search': 'name=Library'})[0]
+
+        product = entities.Product(organization=self.org).create()
+        custom_yum_repo = entities.Repository(product=product,
+                                              content_type='yum',
+                                              url=FAKE_9_YUM_REPO
+                                              ).create()
+
+        # Step 2: Create custom tools, rhel repos and sync them
+        tools_repo, rhel_repo = self._create_custom_rhel_tools_repos(client_os, product)
+        product.sync()
+        product = product.read()
+
+        # Step 3: Create content view and publish it
+        content_view = entities.ContentView(organization=self.org).create()
+        content_view.repository = [custom_yum_repo, tools_repo, rhel_repo]
+        content_view = content_view.update(['repository'])
+        content_view.publish()
+        content_view = content_view.read()
+
+        # Step 4: Create activation key and add subscription.
+        ak = entities.ActivationKey(
+            content_view=content_view,
+            organization=self.org.id,
+            environment=environment
+        ).create()
+        subscription = entities.Subscription(organization=self.org).search(query={
+            'search': 'name={}'.format(product.name)
+        })[0]
+        ak.add_subscriptions(data={
+            'subscription_id': subscription.id})
+
+        # Step 5: Registering Docker Content Host RHEL7
```

Comment:
```suggestion
        # Step 5: Registering Docker Content Host RHEL7
```

please remove above docsting and same for below type of docstring too.